test/functionalTest/cases/0000_log_operation/log_rest.test

```diff
@@ -136,6 +136,7 @@ Date: REGEX(.*)
 
 
 
+
```

Comment:
Not sure why the new way of dealing `_response` requires this .test to add an empty line to work... However, it doesn't seem to be important.